### PR TITLE
FAQ: Revision on punctuation

### PR DESCRIPTION
**Pull request by DeeJayLSP**
This pull request moves some periods to their right place, and adds a missing question mark.

### CODE REVIEW

File: about/faq.rst [hunk 1]
```diff
@@ -9,7 +9,7 @@ Frequently asked questions
 What can I do with Godot? How much does it cost? What are the license terms?
 ----------------------------------------------------------------------------
 
-Godot is `Free and Open-Source Software <https://en.wikipedia.org/wiki/Free_and_open-source_software>`_ available under the `OSI-approved <https://opensource.org/licenses/MIT>`_ MIT license. This means it is free as in "free speech" as well as in "free beer."
+Godot is `Free and Open-Source Software <https://en.wikipedia.org/wiki/Free_and_open-source_software>`_ available under the `OSI-approved <https://opensource.org/licenses/MIT>`_ MIT license. This means it is free as in "free speech" as well as in "free beer".
```
Comment (mortarroad) [inline about/faq.rst:12]: Actually, the period does belong within the quotes in English. It was already correct.

File: about/faq.rst [hunk 2]
```diff
@@ -247,7 +247,7 @@ functionality to officially supported add-ons in future versions of Godot.
 In terms of binary size, this also has the advantage of making you pay only for
 what you actually use in your project. (In the meantime, you can
 :ref:`compile custom export templates with unused features disabled <doc_optimizing_for_size>`
-to optimize the distribution size of your project.)
+to optimize the distribution size of your project).
```
Comment (mortarroad) [inline about/faq.rst:250]: Again, this was already correct. The sentence was started within the parens, so it also ends within them.

File: about/faq.rst (hunk 1 above)
Comment (mhilbrunner) [inline about/faq.rst:12]:
```suggestion
Godot is `Free and Open-Source Software <https://en.wikipedia.org/wiki/Free_and_open-source_software>`_ available under the `OSI-approved <https://opensource.org/licenses/MIT>`_ MIT license. This means it is free as in "free speech" as well as in "free beer."
```

File: about/faq.rst [hunk 3]
```diff
@@ -18,7 +18,7 @@ In short:
 
 All the contents of this accompanying documentation are published under
 the permissive Creative Commons Attribution 3.0 (`CC-BY 3.0 <https://creativecommons.org/licenses/by/3.0/>`_) license, with attribution
-to "Juan Linietsky, Ariel Manzur and the Godot Engine community."
+to "Juan Linietsky, Ariel Manzur and the Godot Engine community".
```
Comment (mhilbrunner) [inline about/faq.rst:21]:
```suggestion
to "Juan Linietsky, Ariel Manzur and the Godot Engine community."
```

File: about/faq.rst (hunk 2 above)
Comment (mhilbrunner) [inline about/faq.rst:250]:
```suggestion
to optimize the distribution size of your project.)
```

**Comment by mhilbrunner** [comment]
Merged the fixed, left switching the correct periods around out :) Thanks!

**Comment by Calinou** [comment]
Cherry-picked to the `3.3` and `stable` branches.

**Pull request merged**